src/controllers/notifications.js

```diff
@@ -86,16 +90,47 @@ exports.post = function * postNotification () {
   try {
     yield model.processNotification(notification, this.ledgers, this.config)
   } catch (e) {
-    if (!(e instanceof UnacceptableExpiryError)) {
+    if (
+      (e instanceof AssetsNotTradedError) ||
```

Comment:
Could these be put under a parent class so there is just one class to check for?